docs/development/language.md

```diff
@@ -0,0 +1,68 @@
+presentation of language entries because the `language_module_ID` is not taken into consideration when ILIAS inserts 
+language entries into the HTML output.
+
+3. New components MUST use the `object_ID` for the `language_module_ID` as defined in the related module.xml or 
+service.xml. The `language_module_ID` MUST also be used as a prefix for the variable names in this language module, 
+e.g.:
+
+        `frm#:#frm_new_posting#:#New Posting`
+
+4. Each language file contains one block with language_module_ID `common`. Entries of this block start with 
+`common#:#`. The language_module_ID `common` MUST only be used for language entries that are used by various 
+components and in combined contexts. Because this block is always read from the database into memory for each 
+request, the use of new `common` variables SHOULD be minimised.
+ 
+5. To keep the language files maintainable and facilitate translation and creation of new language versions, 
+the amount of language entries should be as low as possible. Therefore, language entries that are no longer 
+used in ILIAS due to refactorings or changes in the code MUST be removed from the English language file. 
+Whenever possible, existing language entries SHOULD be reused and probably moved to the 'common' module to 
+avoid multiple entries of the same meaning.
+
+6. The English language file is the master language file. New variables MUST be added at least to this file, 
+since we synchronise the variables when preparing a new ILIAS release. If a variable exists in a file of another 
+language but not in the English one, the entry will be removed from the file.
+
+# Additional Information
+Adding new entries into language files will not make them available automatically in the user interface. You need to 
+refresh the languages by executing Administration » Languages » Refresh Languages in your ILIAS installation.
+ 
+The global language object `$lng`, an instance of class `ilLanguage`, provides methods to access these strings in the 
+language of the user currently logged in. This is done by using the functions `loadLanguageModule()` and `txt()`.
+
+        $lng->loadLanguageModule("frm");
+        $tpl->setVariable("TEXT", $lng->txt("frm_new_posting"));    
+
+# Roles
+The language handling process in ILIAS knows four distinct roles:
+
+* The *First language maintainer* is managing the language handling process and fixing or delegating language bugs. The
+first maintainer has to be notified about newly introduced languages and changes of the language version maintainers.
+* The *Second language maintainer* is responsible for the language component.
+* *Developers* are adding new language entries to their components, modifying them or remove them from the code.
+* *Language version maintainers* are volunteers and take care of a specific language version and translating new entries 
```

Comment:
Same here, maybe `taking care` and `translating`?

Reply:
Same here:
```suggestion
* *Language version maintainers* are volunteers, take care of a specific language version and translate new entries.
```

Reply:
Fully agree!